### ISSUE TITLE
eslint 1.7.2 breaks `indent` rule on multi-lined arguments

### ISSUE THREAD

**Issue by adrienverge** [eslint/eslint#4224]
Since yesterday (bump to 1.7.2) our CI complains because the following code (adapted) does not pass `indent` rule anymore:

```js
example(
    precedingArg, function () {
        console.log('example');
    }
);
```

```
  3:9   error    Expected indentation of 4 space characters but found 8  indent
```

This is introduced by commit 9488faf, which fixed  #4165 and #4164 but apparently introduced this.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Duplicate https://github.com/eslint/eslint/issues/4174
